src/Interfaces.test.ts

```diff
@@ -1,34 +1,46 @@
 /* tslint:disable:no-relative-imports */
 import { describe } from 'riteway'
-import { ClaimType, Identity, isClaim, isIdentity, Work } from './Interfaces'
+import { ClaimType, Identity, isClaim, isIdentity, isWork, Work } from './Interfaces'
+
+const signatureBlock = {
+  '@graph': {
+    '@type': 'sec:Ed25519Signature2018',
+    'http://purl.org/dc/terms/created': {
+      '@type': 'http://www.w3.org/2001/XMLSchema#dateTime',
+      '@value': '2018-09-05T20:19:20Z',
+    },
+    'http://purl.org/dc/terms/creator': {
+      '@id': 'po.et://entities/1bb5e7959c7cb28936ec93eb6893094241a5bc396f08845b4f52c86034f0ddf8',
+    },
+    'https://w3id.org/security#jws':
+      'eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..neh08G8ljeqeEnqqPS06tgV7OGrlVRDa6ZfidRy8cGztWX9QbzwYz5VXtdO8SLy-H8alKeRiwGy67Q_QyEtuCg',
+  },
+}
 
 const TheRaven: Work = {
   id: '1bb5e7959c7cb28936ec93eb6893094241a5bc396f08845b4f52c86034f0ddf8',
-  publicKey: '02badf4650ba545608242c2d303d587cf4f778ae3cf2b3ef99fbda37555a400fd2',
-  signature:
-    '3045022100e020a7ffeffa5d40ffde618c6c861678e38de69fd377028ec57ad93893883b3702201f085284a9064bab7e1cd39349e65d136d8f67e4b6b897c3e7db6b400ed91034',
+  issuer: 'data:,1bb5e7959c7cb28936ec93eb6893094241a5bc396f08845b4f52c86034f0ddf8',
   type: ClaimType.Work,
-  created: '2017-11-13T15:00:00.000Z',
-  attributes: {
+  issuanceDate: '2017-11-13T15:00:00.000Z',
+  claim: {
     name: 'The Raven',
     author: 'Edgar Allan Poe',
     tags: 'poem',
     dateCreated: '',
     datePublished: '1845-01-29T03:00:00.000Z',
-    text: 'Once upon a midnight dreary...',
   },
+  'sec:proof': signatureBlock,
 }
 
 const Me: Identity = {
   id: '1bb5e7959c7cb28936ec93eb6893094241a5bc396f08845b4f52c86034f0ddf8',
-  publicKey: '02badf4650ba545608242c2d303d587cf4f778ae3cf2b3ef99fbda37555a400fd2',
-  signature:
-    '3045022100e020a7ffeffa5d40ffde618c6c861678e38de69fd377028ec57ad93893883b3702201f085284a9064bab7e1cd39349e65d136d8f67e4b6b897c3e7db6b400ed91034',
+  issuer: 'did:po.et:1bb5e7959c7cb28936ec93eb6893094241a5bc396f08845b4f52c86034f0ddf8',
```

Comment:
Is there a doc describing our use of DIDs, or is this just an example of what could happen?

Reply:
Oops I need to remove that. We are not yet supporting DIDs